src/main/java/org/opentripplanner/middleware/utils/ItineraryUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.middleware.utils;
+
+import org.opentripplanner.middleware.otp.response.Itinerary;
+import org.opentripplanner.middleware.otp.response.Leg;
+
+public class ItineraryUtils {
```

Comment:
Should the class have JavaDoc?

Reply:
Added in https://github.com/ibi-group/otp-middleware/pull/91/commits/43425abecb53ee8d559c0d7c75062e6c84b7f03d.